recipes/cmake/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: cmake
+  version: 3.5.0
+
+source:
+  url: https://cmake.org/files/v3.5/cmake-3.5.0.tar.gz  # [not win]
+  fn: cmake-3.5.0.tar.gz # [not win]
+  url: https://cmake.org/files/v3.5/cmake-3.5.0-win32-x86.zip  # [win]
+  fn: cmake-3.5.0-win32-x86.zip # [win]
+
+
+test:
+  commands:
+    - cmake --version
+
+about:
+  home: http://www.cmake.org/
+  license:  3-clause BSD
+  license_family: BSD
+  summary: CMake is an extensible, open-source system that manages the build process
```

Comment:
We need to add the "extra/recipe-maintainers" section. Feel free to add me there as well. (The more the merrier :wink: )